_events/2023/2023-06-community-call.md

```diff
@@ -0,0 +1,21 @@
+---
+title: June 2023 Community Call
+expires: 2023-06-10
+event_date: "June 9, 2023"
+layout: event
+duration: 60
+repeated: false
+category: community-call
+time:
+    - - start: 2023-06-09T18:00:00Z
+        end: 2023-05-11T19:00:00Z
```

Comment:
End date needs to be updated!

Reply:
Oops sorry about that! This is the first one of these I've done, and I overlooked that.